assets/js/pages/SettingsPage/hooks.test.jsx

```diff
@@ -319,3 +327,144 @@ describe('useApiKeySettings', () => {
     expect(hookResult.current.apiKey).toBe('new_api_key');
   });
 });
+
+describe('useAlertingSettings', () => {
+  const saveAlertingSettings = alertingSettingsSaveRequestFactory.build();
+  const saveAlertingData =
+    alertingSettingsSaveRequestToApiData(saveAlertingSettings);
+
+  const settingsFromSaveSettings = (alertingSettings) =>
+    flow(
+      pick([
+        'alertingEnabled',
+        'smtpServer',
+        'smtpPort',
+        'smtpUsername',
+        'senderEmail',
+        'recipientEmail',
+      ]),
+      set('enforcedFromEnv', false)
+    )(alertingSettings);
+
+  const dataFromSaveData = (alertingData) =>
+    flow(
+      pick([
+        'enabled',
+        'smtp_server',
+        'smtp_port',
+        'smtp_username',
+        'sender_email',
+        'recipient_email',
+      ]),
+      set('enforced_from_env', false)
+    )(alertingData);
```

Comment:
Besides naming not being immediately clear, to me at least, I am wondering: since these two functions are always called with the same constant arguments, couldn't they be just two more constants? 👀 

Or am I missing something?

```suggestion
  const fooNeedsBetterName = flow(
      pick([
        'alertingEnabled',
        'smtpServer',
        'smtpPort',
        'smtpUsername',
        'senderEmail',
        'recipientEmail',
      ]),
      set('enforcedFromEnv', false)
    )(saveAlertingSettings);

  const barNeedsBetterName = flow(
      pick([
        'enabled',
        'smtp_server',
        'smtp_port',
        'smtp_username',
        'sender_email',
        'recipient_email',
      ]),
      set('enforced_from_env', false)
    )(saveAlertingData);
```